cmd/poller/collector/collector.go

```diff
@@ -281,6 +281,13 @@ func (me *AbstractCollector) Start(wg *sync.WaitGroup) {
 				continue
 			}
 
+			if me.Schedule.IsStandBy() && !me.Schedule.IsTaskStandBy(task) {
```

Comment:
IsTaskStandBy makes the same check, so we could simplify this to 
```go
if !me.Schedule.IsTaskStandBy(task) {
```
or we could flip it around to this which maybe reads a little better?

`if me.Schedule.ShouldSkip(task) {`

Reply:
Both are different. Is !TaskStandBy will be true in all happy cases. Hence added additional standbycheck outside

Reply:
removed an redundant check from IsTaskStandBy.